### PR TITLE
Feat/healthcheck

### PR DESCRIPTION
**Pull request by dvilelaf**
* Implements a healthcheck

### CODE REVIEW

**Review by Adamantios — APPROVED** [review]

File: run_service.sh
[the remark this answers is not in the document]
Comment (dvilelaf) [reply run_service.sh:1]: I had to add this to my .env:
`STORE_PATH='/tmp/'`

File: run_service.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+REPO_PATH=$PWD
+
```
Comment (Adamantios) [inline run_service.sh:4]:
```suggestion

export STORE_PATH='/tmp/'

REPO_PATH=$PWD

```

File: run_service.sh
[the remark this answers is not in the document]
Comment (Adamantios) [reply run_service.sh:1]: Would https://github.com/valory-xyz/trader/pull/262#discussion_r1619052062 work?

[the remark this answers is not in the document]
Comment (dvilelaf) [reply run_service.sh:1]: Yes, accepting it.

**Pull request merged**